### PR TITLE
Modified for new Booster and Workflow management FLP

### PR DESCRIPTION
**Pull request by ArchanaShukla**

### CODE REVIEW

File: tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md [hunk 1]
```diff
@@ -10,19 +10,21 @@ primary_tag: products>sap-cloud-platform
 
 ## Details
 ### You will learn
-  - How to enable and configure workflow, business rules and process visibility services
+  - How to enable and configure workflow, business rules, process visibility and process flexibility capabilities
 
-Digital process automation for live processes is a family of services to automate business processes, manage decision logic and provide end-to-end visibility in your processes.
+**SAP Cloud Platform Workflow Management** is a cloud offering that helps you digitize structured processes with workflows and decision logic, flexibly extend and adapt business processes running in your packaged applications, manage process variants and decision logic, and monitor end-to-end process metrics in real time.
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md:15]: This sentence is very long, which makes it hard to read. Can you please break it up into more sentences?

File: tutorials/cp-starter-ibpm-employeeonboarding-3-workflow/cp-starter-ibpm-employeeonboarding-3-workflow.md [hunk 2]
```diff
@@ -41,15 +41,15 @@ Download **cf-onboarding-sample.zip**  project from [GitHub](https://github.com/
...
+5. You will see that the **cf-onboarding-sample** project is imported into your workspace.
 
     ![Workflow Added](sampleworkflowadded.png)
 
     > This sample workflow has:
 
     > - **Business Rules** integrated into it. You can find the integration as part of service task with name `Determine Equipment`.
 
-    > - **User Tasks** for confirmation of the equipment selected via business rules and for the approval of equipment. You can find these user tasks with the names `Confirm and Change Equipment` and `Approve Equipment` respectively.  The user interface associated with these user tasks have been built as SAPUI component and you can find these modules bundled under `confirmEquip` and `approveEquip` respectively in the same multi target application.
+    > - **User Tasks** for confirmation of the equipment selected via business rules and for the approval of equipment. You can find these user tasks with the names `Confirm and Change Equipment`, `Approve Equipment` and `Accept Workplace` respectively.  The user interface associated with these user tasks have been built using Workflow Form and you can find these under **forms** folder in the same multi target application.
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-3-workflow/cp-starter-ibpm-employeeonboarding-3-workflow.md:52]:
```suggestion
    > - **User Tasks** for confirmation of the equipment selected via business rules and for the approval of equipment. You can find these user tasks with the names `Confirm and Change Equipment`, `Approve Equipment` and `Accept Workplace` respectively.  The user interface associated with these user tasks has been built using Workflow Form and you can find these under **forms** folder in the same multi target application.
```

File: tutorials/cp-starter-ibpm-employeeonboarding-5-run/cp-starter-ibpm-employeeonboarding-5-run.md [hunk 3]
```diff
@@ -172,7 +172,7 @@ Now, you will run the workflow and see how the new events and data are automatic
 
     ![View Scenario4](viewscenario_4.png)
 
-Congratulations. With this, you have successfully completed the mission on **Automating and Extending Employee Onboarding** process using Workflow Management services. In this mission, you learnt about SAP Cloud Platform Workflow, Business Rules and Process Visibility and how these services come together to help you automate the process and decisions, together with providing out-of-box visibility into the workflows.
+Congratulations. With this, you have successfully completed the mission on **Automating and Extending Employee Onboarding** process using Workflow Management services. In this mission, you learnt about Workflow, Business Rules and Process Visibility capabilities of **SAP Cloud Platform Workflow Management** and how they come together to help you automate the process and decisions, together with providing an out-of-box visibility into the workflows.
```
Comment (MichaelCzcz) [inline tutorials/cp-starter-ibpm-employeeonboarding-5-run/cp-starter-ibpm-employeeonboarding-5-run.md:175]:
```suggestion
Congratulations. With this, you have successfully completed the mission on **Automating and Extending Employee Onboarding** process using Workflow Management services. In this mission, you learned about Workflow, Business Rules and Process Visibility capabilities of **SAP Cloud Platform Workflow Management** and how they come together to help you automate the process and decisions, together with providing an out-of-box visibility into the workflows.
```

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/cp-starter-ibpm-employeeonboarding-5-run/cp-starter-ibpm-employeeonboarding-5-run.md (hunk 3 above)
Comment (ArchanaShukla) [reply tutorials/cp-starter-ibpm-employeeonboarding-5-run/cp-starter-ibpm-employeeonboarding-5-run.md:175]: Modified

**Pull request merged**